src/macros.jl

```diff
@@ -577,6 +577,32 @@ macro subset(x, args...)
     esc(subset_helper(x, args...))
 end
 
+function rsubset_helper(x, args...)
+    exprs, outer_flags = create_args_vector(args...)
+    if outer_flags[Symbol("@byrow")][] == true
```

Comment:
```suggestion
    if outer_flags[Symbol("@byrow")][]
```
(Same in similar cases.)